src/main/java/org/myrobotlab/service/InMoovHand.java

```diff
@@ -128,12 +110,7 @@ public void bird() {
   @Override
   public void broadcastState() {
     // notify the gui
-    thumb.broadcastState();
-    index.broadcastState();
-    majeure.broadcastState();
-    ringFinger.broadcastState();
-    pinky.broadcastState();
-    wrist.broadcastState();
+    Runtime.broadcastStates();

```

Comment:
why is this runtime broadcast state? shouldn't it be super.broadcastState()  .. perhaps this should just be removed from here and rely on the base class.